packages/now-cli/src/commands/deploy/legacy.ts

```diff
@@ -269,6 +270,17 @@ export default async function main(
     paths = [process.cwd()];
   }
 
+  if (argv._[0] === homedir()) {
```

Comment:
Consider turning this into a function so it can be used in both v1 and v2:

```ts
async function shouldDeployDir(argv0, output) {
  let yes = true;
  if (argv0 === homedir()) {
    yes = await promptBool('You are deploying your home directory. Do you want to continue?');
    if (!yes) {
      output.log('Aborted');
    }
  }
  return yes;
}

// then the only repeated code you need is the following:
if (!(await shouldDeployDir(argv._[0], output))) {
  return 0;
}
```

Reply:
will do 🙌